data/hai/unfettered side missions.txt

```diff
@@ -432,6 +437,7 @@ mission "Hai Legacy 3"
 			has "unfettered: beaten extra bugs"
 	on complete
 		clear "unfettered: beaten the bugs"
+		clear "old reputation: Hai (Unfettered)"
```

Comment:
This doesn't clear the condition if you failed the mission. Not sure if it can be declined.

Reply:
should be cleared in most situations I can think of now
~~honestly if an odd condition remains its not the end of the world~~
but yes, I do agree we should remove them when we can